Source/RiveEditor/Private/Factories/RiveWidgetFactory.cpp

```diff
@@ -199,8 +199,8 @@ namespace
 	{
 		if (const UContentBrowserAssetContextMenuContext* Context = UContentBrowserAssetContextMenuContext::FindContextWithAssets(MenuContext))
 		{
-			TArray<URiveFile*> RiveFiles = Context->LoadSelectedObjects<URiveFile>();
-			for (URiveFile* RiveFile : RiveFiles)
+			TArray<URiveFile*> URiveFiles = Context->LoadSelectedObjects<URiveFile>();
```

Comment:
Just as a stylistic comment - Typically the class prefix (U, F, E, etc. that is required for Unreal-based types) aren't used in Variable Names themselves.

Reply:
Copy/Paste error. Good catch.